src/components/SearchGridFilter.vue

```diff
@@ -92,41 +169,108 @@ export default {
         shouldNavigate: true,
       });
     },
-    onUpdateSearchByCreator() {
-      this.$store.dispatch(TOGGLE_FILTER, {
-        filterType: 'searchBy',
+    onClearFilters() {
```

Comment:
Please don't change the order of methods. Can you revert them?

Reply:
Done.